contracts/contracts/ProjectRegistry.sol

```diff
@@ -0,0 +1,128 @@
+// SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+/**
+ * @title ProjectRegistry
+ * @notice todo
+ * @dev todo
+ */
+contract ProjectRegistry {
+    // Types
+
+    // The pointer to the external metadadata composed by protocol and pointer
+    struct MetaPtr {
+        uint256 protocol;
+        string pointer;
+    }
```

Comment:
wanna move this to it's own .sol and import it ? 
https://github.com/gitcoinco/grants-round/blob/main/packages/contracts/contracts/utils/MetaPtr.sol

Reply:
@thelostone-mc good point, I'll do it!